.github/workflows/odoo-ci-cd.yml

```
@@ -39,3 +38,128 @@ jobs:
+
+          # Export for use in later steps
+          echo "docker_instances=$DOCKER_INSTANCES" >> "$GITHUB_OUTPUT"
+
+      - name: Run Graph-SYNC Project
+        run: |
+            docker_instances="${{ steps.docker_instances.outputs.docker_instances }}"
+            echo "docker_instances: $docker_instances"
+            docker run --network odoo_network \
+              -e DOCKER_INSTANCES="$docker_instances" \
+              ghcr.io/gldzzpro/graph_sync:latest
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)